src/wcm.cpp

```diff
@@ -390,7 +459,18 @@ OptionWidget::OptionWidget(Option *option) : Gtk::Box(Gtk::ORIENTATION_HORIZONTA
     {
         if (option->str_labels.empty())
         {
-            auto entry = std::make_unique<Gtk::Entry>();
+            std::unique_ptr<Gtk::Entry> entry;
+            if (option->name == "xkb_layout")
+            {
+                entry = std::make_unique<LayoutsEntry>();
+            } else if (option->name == "xkb_model")
+            {
+                entry = std::make_unique<XkbModelEntry>();
+            } else
+            {
+                entry = std::make_unique<Gtk::Entry>();
+            }
+
             entry->set_text(wf_option->get_value_str());
```

Comment:
xkb_layout option supports a comma separated list of layouts according to [the wiki](https://github.com/WayfireWM/wayfire/wiki/Configuration#inputxkb_layout). Are you still able to do this with this patch?

Reply:
Yes, this patch only adds suggestions to the context menu of `Gtk::Entry`. It looks like this:
![view_2023-03-20_11-07-20](https://user-images.githubusercontent.com/50048443/226281522-e62b7772-fb7f-4e0d-bc7f-e04b023fc8f1.png)

And context menu looks like this
![screenshot_2023-03-20-110947](https://user-images.githubusercontent.com/50048443/226281762-31170b54-0409-48f9-ba0e-28790327aaa9.png)

Clicking on one of the layouts appends the layout to the comma separated list (e.g. clicking "au — English (Australian)" appends ", au").